utils/albumentations16.py

```diff
@@ -116,6 +116,10 @@ def __init__(self, always_apply=False, p=0.5):
 
     def apply(self, img, **params):
         return cv2.normalize(img, None, 0, 255, cv2.NORM_MINMAX, cv2.CV_8U)
+    def get_transform_init_args_names(self):
+        # This transform does not require any initialization arguments for logging/serialization
```

Comment:
why not copying the docstring of the abstract function?
docstrings are exactly for that, explaining what the function does, not traditional comments 🙅 

https://github.com/albumentations-team/albumentations/blob/5eb2da80398ed7a25d0e67f97adb0f77e4206f22/albumentations/core/transforms_interface.py#L285

```python
def get_transform_init_args_names(self) -> tuple[str, ...]:
        """Returns names of arguments that are used in __init__ method of the transform."""
```